nuxt.config.ts

```diff
@@ -0,0 +1,146 @@
+import svgLoader from 'vite-svg-loader'
+
+export default defineNuxtConfig({
+    ssr: false,
+    vite: {
+        plugins: [
+            svgLoader({
+                defaultImport: 'component',
+                svgoConfig: {
+                    multipass: true,
+                },
+            }),
+        ],
+    },
+    imports: {
+        autoImport: true,
+    },
```

Comment:
Ideally this is set to false, Otherwise you have no control on what component Nuxt is actually importing.

Reply:
Correct, it's not needed, will take it out